docs/pages/apis/rest.fr.mdx

```diff
@@ -0,0 +1,123 @@
+
+```http
+GET https://api.potterdb.com/v1/characters?page[number]=2
+```
+
+## Ransack
+
+Notre API REST supporte une fonctionnalité de filtre avancé et de tri en utilisant [Ransack](https://activerecord-hackery.github.io/ransack/).
+
+### Filtrer
+
+Vous pouvez filtrer une requête en ajoutant le mot-clef `filter` en paramètre de requête. Par exemple, pour filter les personnages par nom, vous utiliserez le prédicat `name_cont` :
+
+```http
+GET https://api.potterdb.com/v1/characters?filter[name_eq]=Weasley
+```
+
+Ceci vous retournera tous les personnages qui portent le nom "Weasley".
+
+<details>
+  <summary>Agrandir pour voir tous les prédicats de filtres disponibles</summary>
+  - `eq`: égal - `eq_any`: égal à n'importe lequel des choix - `eq_all`: égal à tous les choix
+  - `not_eq`: n'est pas égal à - `not_eq_any`: n'est égal à n'importe lequel des choix - `not_eq_all`: n'est égal à aucun des choix 
+  - `matches`: correspond - `matches_any`: correspond à n'importe lequel des choix - `matches_all`: correspond à tous les choix 
+  - `does_not_match`: ne correspond pas - `does_not_match_any`: ne correspond pas à n'importe lequel des choix - `does_not_match_all`: ne correspond à aucun des choix
+  - `lt`: est plus petit que - `lt_any`: est plus petit que n'importe lequel des choix - `lt_all`: est plus petit que tous les choix
+  - `lteq`: est plus petit ou égal à - `lteq_any`: est plus petit ou égal à n'importe  lequel des choix - `lteq_all`: est plus petit ou égal à tous les choix
+  - `gt`: est plus grand que - `gt_any`: est plus grand que n'importe lequel des choix - `gt_all`: est plus grand que tous les choix
+  - `gteq`: est plus grand ou égal à - `gteq_any`: est plus grand ou égal à n'importe lequel des choix - `gteq_all`: est plus grand ou égal à tous les choix
+  - `in`: dans - `in_any`: dans n'importe lequel des choix - `in_all`: dans tous les choix
+  - `not_in`: n'est pas dans - `not_in_any`: n'est pas dans n'importe lequel des choix - `not_in_all`: n'est dans aucun des choix
+  - `cont`: contient - `cont_any`: contient n'importe lequel des choix - `cont_all`: contient tous les choix
+  - `not_cont`: ne contient pas - `not_cont_any`: ne contient pas n'importe lequel des choix - `not_cont_all`: ne contient aucun des choix
+  - `start`: débute par -`start_any`: débute par n'importe lequel des choix - `start_all`: débute par tous les choix
+  - `not_start`: ne débute pas par - `not_start_any`: ne débute pas par n'importe lequel des choix - `not_start_all`: ne débute par aucun des choix
+  - `end`: termine par - `end_any`: termine par n'importe lequel des choix - `end_all`: termine par tous les choix
+  - `not_end`: ne termine pas par - `not_end_any`: ne termine pas par n'importe lequel des choix - `not_end_all`: ne termine par aucun des choix
+  - `'true'`: est vrai - `'false'`: est faux - `present`: est présent - `blank`: est vide - `'null'`: est nul - `not_null`: n'est pas nul
```

Comment:
Could you please clean this a bit up? E.g. every `-` should start in a new line. Looks a bit chaotic rn.

Reply:
Absolutely ! I didn't dare doing this because the english file had only 2 new lines so I did... something between the two. I'll clean it with pleasure